contest/views.py

```diff
@@ -166,6 +179,9 @@ def delete(request, cid):
     if can_delete_contest(request.user, contest):
         deleted_cid = contest.id
         contest.delete()
+        message = 'Contest %s deleted!' % \
```

Comment:
This is for telling the user the result of deleting contest
not for warning in advance